### PR TITLE
Insteon Scene Automation

### PR DESCRIPTION
**Pull request by teharris1**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#24765

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

<a href="https://gitpod.io/#https://github.com/home-assistant/home-assistant.io/pull/9692"></a>

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_components/insteon.markdown [hunk 1]
```diff
@@ -258,14 +258,33 @@ light:
     address: 1a2b3c
 ```
 
+### {% linkable_title INSTEON Scenes %}
+
+Trigger an INSTEON scene on or off is done via automations. Two services are provied to support this feature:
```
Comment (MartinHjelmare) [inline source/_components/insteon.markdown]: provided

**Comment by klaasnicolaas** [comment]
You still have to resolve a merge conflict 😉

**Review by klaasnicolaas — APPROVED** [review]

**Comment by klaasnicolaas** [comment]
Time to merge it. Thnx for the PR! 👍

**Pull request merged**